apps/studio/components/interfaces/Home/Connect/Connect.tsx

```diff
@@ -170,7 +170,7 @@ const Connect = () => {
             </span>
           </Button>
         </DialogTrigger>
-        <DialogContent className={cn('sm:max-w-5xl p-0')}>
+        <DialogContent className={cn('sm:max-w-5xl p-0 bg-surface-100')}>
```

Comment:
this isn't the dialog background.
I'll do a follow up PR, think we need a new css class for Dialog backgrounds